app/bundles/CoreBundle/Entity/TranslationEntityTrait.php

```diff
@@ -16,12 +16,12 @@ trait TranslationEntityTrait
     public $languageSlug;
 
     /**
-     * @var ArrayCollection
+     * @var mixed
```

Comment:
This won't work? Similar to other changes in this file
```suggestion
     * @var ArrayCollection<TranslationEntityInterface>
```